client/src/components/Atoms/FileUpload.tsx

```diff
@@ -1,154 +1,190 @@
 import React from 'react'
-import { FileInput, Button, Colors, ProgressBar } from '@blueprintjs/core'
+import { H5, FileInput, Button, Callout, AnchorButton } from '@blueprintjs/core'
 import { useForm } from 'react-hook-form'
 import styled from 'styled-components'
-import StatusTag from './StatusTag'
+import StatusTag, { StatusTagWithProgress } from './StatusTag'
 import { IFileUpload } from '../useFileUpload'
 import AsyncButton from './AsyncButton'
+import { assert } from '../utilities'
 
-const ErrorP = styled.p`
-  margin-top: 10px;
-  color: ${Colors.RED3};
+const Row = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  &:not(:last-child) {
+    margin-bottom: 10px;
+  }
+`
+
+const buttonAndTagWidth = '115px' // Wide enough for the longest text
+
+const FileStatusTag = styled(StatusTag)`
+  width: ${buttonAndTagWidth};
+  text-align: center;
+`
+
+const FileStatusTagWithProgress = styled(StatusTagWithProgress)`
+  width: ${buttonAndTagWidth};
+  text-align: center;
```

Comment:
Minor: This `text-align: center;` could be applied universally within `StatusTag`, since I imagine almost any case that wants to increase the width will want the text centered

Reply:
Though on second thought I also see the benefit of applying it here when we actually need it since its purpose may not be clear if added to the original base-width `StatusTag`

Reply:
I think I agree with both perspectives haha. I'll probably just leave it unless you have a strong leaning